### ISSUE TITLE
Project.xml-specified background color overrides FlxG.camera.bgColor on native targets on next

### ISSUE THREAD

**Issue by starry-abyss** [HaxeFlixel/flixel#1874]
- **Flixel version:** dev, 66b1522
- **OpenFL version:** openfl3, b70de54
- **Lime version:** develop, 6111cf4
- **Affected targets:** neko, windows

________________________________

Hello, let's continue digging issues :-)

Note: I use dots (...) in code snippets to omit irrelevant parts for readability.
Take any simple project, modify Project.xml (green background):

```haxe
	<window ... background="#00ff00" ... />
	....
	<set name="next"/>
	<haxelib name="flixel"/>
	....
```

Modify to PlayState.hx (red background):

```haxe
override public function create():Void
{
	....
	FlxG.camera.bgColor = FlxColor.RED;
}
```
________________________________

Then when running on flash, you'll get red background (this is correct and was so on all targets in older library versions on legacy mode).
When running on neko and windows, though, background is green.

**Comment by sruloart** [comment]
There was an OpenFL issue over at "next" where the native Flixel background sprite didn't fill the screen, thus you could only see the renderer preset background.

https://github.com/HaxeFlixel/flixel/issues/1793
https://github.com/HaxeFlixel/flixel/issues/1646

Weirdly enough, testing with the current releases openfl/lime(3.6.1/2.9.1) I don't seem to have that problem anymore on native targets (well, Win/Neko), but it's still relevant for html5 (which has an open issue already).

tl;dr That's an OpenFL bug, and I do hope there's no regression on its dev branches because it seems to be partially fixed.

**Comment by starry-abyss** [comment]
@sruloart I don't see it on 3.6.1/2.9.1 too, but these versions are unusable for me on 'next' because of audio crashes :-)

**Comment by sruloart** [comment]
A quick workaround would be making background tiles on your game's tilemap instead of relying on the Flixel background sprite.

**Comment by Gama11** [comment]
I think this is another instance of #1793 and should now be fixed.
